carapace-server/src/main/resources/conf/server.dynamic.properties

```diff
@@ -54,7 +54,8 @@ route.100.action=cache-if-possible
 default.action.notfound=not-found
 default.action.internalerror=internal-error
 
-
+# Custom X-Proxy-Path header
+mapper.debug=true
```

Comment:
Should it be debug by default?

Reply:
In production environment is enabled, it's is very useful even in dev/debug environment.